dashboard/app/serializers/api/v1/pd/workshop_serializer.rb

```diff
@@ -1,36 +1,6 @@
-# == Schema Information
-#
-# Table name: pd_workshops
-#
-#  id                  :integer          not null, primary key
-#  organizer_id        :integer          not null
-#  location_name       :string(255)
-#  location_address    :string(255)
-#  processed_location  :text(65535)
-#  course              :string(255)      not null
-#  subject             :string(255)
-#  capacity            :integer          not null
-#  notes               :text(65535)
-#  section_id          :integer
-#  started_at          :datetime
-#  ended_at            :datetime
-#  created_at          :datetime
-#  updated_at          :datetime
-#  processed_at        :datetime
-#  deleted_at          :datetime
-#  regional_partner_id :integer
-#  on_map              :boolean
-#  funded              :boolean
-#
-# Indexes
-#
-#  index_pd_workshops_on_organizer_id         (organizer_id)
-#  index_pd_workshops_on_regional_partner_id  (regional_partner_id)
-#
-
```

Comment:
This was already out of sync with the model, and doesn't appear in all our serializers, so it seemed cleanest to just remove it.